dev015-making-apps-more-powerful/04-custom-data-final/add-custom-data/PublicAppUsingDeviceCodeFlow.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Microsoft.Identity.Client;
```

Comment:
Implementation of the base class, mainly by moving common methods / member from MyInformation